MovieApp/Screens/Details/DetailsViewController.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  DetailsViewController.swift
+//  MovieApp
+//
+//  Created by MinhDev on 10/6/21.
+//
+
+import UIKit
+
+class DetailsViewController: UIViewController, Storyboarded {
+
+	override func viewDidLoad() {
+	super.viewDidLoad()
+	}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Code4Fun-Group/Movie-App/blob/b9bc53c4ca90b499597b082b0a499ecbaeee8d75/MovieApp/Screens/Details/DetailsViewController.swift#L14"></span>Override methods which only call super can be removed</td>
    </tr>
  </tbody>
</table>